src/mosaic.mjs

```diff
@@ -226,12 +254,13 @@ async function mosaic512px(z, x, y, filters = {}) {
     },
   })
     .composite(
-      tiles
-        .filter((tile) => !tile.empty())
-        .map((tile) => {
-          return { input: tile.image.buffer, top: 0, left: 0 };
-        })
+      filteredTiles.map(({ tile }) => {
+        return { input: tile.image.buffer, top: 0, left: 0, blend: "over" };
```

Comment:
please share some pictures how it looks like

Reply:
<img width="1028" alt="Screenshot 2024-08-28 at 17 48 11" src="https://github.com/user-attachments/assets/98b644ac-0cf8-4161-ade8-f705ca75c207">
<img width="1026" alt="Screenshot 2024-08-28 at 17 39 17" src="https://github.com/user-attachments/assets/15bd2f92-ea7c-4263-8218-6db6502fb6bb">
Looks same

Reply:
Before
<img width="257" alt="image" src="https://github.com/user-attachments/assets/bdad0fa6-6458-47cf-b86e-2e6303c598bd">
After
<img width="258" alt="image" src="https://github.com/user-attachments/assets/4728c965-ac1e-41aa-93a6-e35403afc746">